### PR TITLE
Moved the registration status type update out of the add order item

### PR DESCRIPTION
**Pull request by christopher-kundrat**
process and into the placeOrder process

---

This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/4959)

### CODE REVIEW

File: model/service/OrderService.cfc [hunk 1]
```diff
@@ -540,36 +543,6 @@ component extends="HibachiService" persistent="false" accessors="true" output="f
...
-								} else {
-									// If we have an unexprected waitlister due to event filling before order item was created
-									// check to see if there were any sales that went through. If there were then move the registrant
-									// to a waitlist/deposit order item. If not then change the order item type to deposit and waitlist registrant.
-									if( !salesOnlyFlag && salesCount > 0 ) {
-										registrant.toWaitlistFlag = "1";
-										orderItemsToCreateCount = 2;
-										salesOnlyFlag = true;
-									} else {
-										newOrderItem.setOrderItemType( getTypeService().getTypeBySystemCode('oitDeposit') );
```
Comment (rmarchandTen24) [inline model/service/OrderService.cfc]: @sumitverma @christopher-kundrat  are we removing the deposit logic intentionally?

Comment (sumitverma) [inline model/service/OrderService.cfc]: We shouldn't be. @christopher-kundrat why are we removing waitlist logic?

Comment (christopher-kundrat) [inline model/service/OrderService.cfc]: @sumitverma The waitlisted logic wasn't removed it was just moved to the placeOrder. I'll just have to add the depositOrderItem logic back in.

Comment (rmarchandTen24) [inline model/service/OrderService.cfc]: @christopher-kundrat just want to bump this now that you are back

**Pull request merged**